cves/kernel/CVE-2022-36879.yml

```diff
@@ -371,8 +371,8 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: the fix is centered around reinitializing pointer, thus preventing a value from being dropped twice
```

Comment:
Dumb this one down, what if someone doesn't know what a pointer is?

Reply:
Thanks! I dumbed it down.